webapp/src/containers/PopOver/__test__/__snapshots__/index.test.tsx.snap

```diff
@@ -78,7 +77,6 @@ exports[`Popover Index should check for snapshot 1`] = `
           returnFocusAfterClose={true}
           role="dialog"
           scrollable={false}
-          trapFocus={false}
```

Comment:
why was this removed? I added it because it's failing on me locally. Can you try to re-install packages and try the tests again?

Reply:
Done